src/test/java/jenkins/branch/CustomOrganizationFolderDescriptorTest.java

```diff
@@ -116,17 +116,32 @@ public void insideFolder() throws Exception {
         public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {}
     }
     private static abstract class MockNavigatorDescriptor extends SCMNavigatorDescriptor {
+
         MockNavigatorDescriptor() {
             super(MockNavigator.class);
         }
+
         @Override
         public SCMNavigator newInstance(String name) {
             return new MockNavigator();
         }
+
         @Override
         public String getDisplayName() {
             return "MockNavigator";
         }
+
+        public String getDescription() {
+            return "MockNavigator description";
+        }
+
+        public String getCategoryId() {
+            return "MockNavigator categoryId";
+        }
+
+        public String getIconFilePathPattern() {
+            return "MockNavigator iconfilepathpattern";
+        }
```

Comment:
@jglick you were right [here](https://github.com/jenkinsci/scm-api-plugin/pull/9#discussion_r58587652) but not these changes here, but because of we can not force to use specific versions of `scm-api` and `branch-api`.

Maybe we have here another reason to include both dependencies directly in **github-branch-source-plugin**.